python/ray/experimental/channel.py

```diff
@@ -47,6 +52,25 @@ def _create_channel_ref(
     return object_ref
 
 
+# 1. Writer creates chan = Channel().
```

Comment:
Can you remove this comment? It'd be good to include something like this in the PR description though!

Reply:
Done.